### PR TITLE
Completed stretch goals

### PR DESCRIPTION
**Pull request by MRLGAnker**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/templates/index.html [hunk 1]
```diff
@@ -6,22 +6,61 @@
...
+          </div>
+        </div>
+      {% endif %}
+    {% endfor %}
+  </div>
+  <div class="my-3 p-3 bg-white rounded shadow-sm">
+    <h3 class="border-bottom border-gray pb-2 mb-0">Add Item</h3>
+    <form action="/" method="post">
+      <div class="input-group mb-3">
+        <input type="text" class="form-control" placeholder="Recipient's username" aria-label="Recipient's username" aria-describedby="button-addon2" id="addItem" name="addItem">
```
Comment (Jonesey13) [inline todo_app/templates/index.html:32]: The placeholder here doesn't make sense in this context.

File: todo_app/data/session_items.py [hunk 2]
```diff
@@ -67,3 +67,18 @@ def save_item(item):
...
+
+def remove_item(item):
+    """
+    Updates an existing item in the session. If no existing item matches the ID of the specified item, nothing is saved.
+
+    Args:
+        item: The item to save.
+    """
+    existing_items = get_items()
+    updated_items = [i for i in existing_items if not (i['id'] == item['id'])]
```
Comment (Jonesey13) [inline todo_app/data/session_items.py:80]: I'd replace `not ==` with  `!=` here

**Review by Jonesey13 — COMMENTED** [review]
Also looks great 👍

File: todo_app/templates/index.html (hunk 1 above)
Comment (MRLGAnker) [reply todo_app/templates/index.html:32]: I think this must've been left over from copy & pasting from the bootstrap examples...

File: todo_app/data/session_items.py (hunk 2 above)
Comment (MRLGAnker) [reply todo_app/data/session_items.py:80]: I'll admit to copying that part from somewhere without quite understanding how it works haha..

**Pull request merged**